plugin/oldfilesearch.vim

```diff
@@ -75,6 +75,20 @@ function! s:GetOldFiles(patterns)
         call add(candidates, ffull)
         let oldindex[ffull] = oidx
     endfor
+    " Prepend all regular buffers that are not yet candidates.
+    for l:b in reverse(range(1, bufnr('$')))
+        " skip non-existing and special buffers.
+        if !bufexists(l:b) || !empty(getbufvar(l:b, '&buftype'))
+            continue
+        endif
+        " skip buffers that are already candidates.
+        let bfull = fnamemodify(bufname(l:b), ':p')
```

Comment:
it seems you should check whether `bufname(l:b)` is empty.
`fnamemodify('', ':p')` is converted to current directory.
maybe it is not what we want, isn't it?